docs/sourcemap-api.asciidoc

```diff
@@ -35,7 +35,7 @@ Example source map request
 
 ["source","sh",subs="attributes"]
 ---------------------------------------------------------------------------
-curl -X POST http://127.0.0.1:8200/v1/rum/sourcemaps \
+curl -X POST http://127.0.0.1:8200/assets/v1/sourcemaps \
```

Comment:
I'd expect to have a note about the deprecated URL somewhere in the docs. It is going to be part of a minor release, we should avoid simply removing deprecated but supported URLS.